Assets/Scripts/Game Object/Card/LeftCard.cs

```diff
@@ -0,0 +1,8 @@
+public class LeftCard : Card
```

Comment:
I think it would be good for the Card subclass names to relate to their functionality.